lib/bump/cli/commands/base.rb

```diff
@@ -18,9 +18,12 @@ def post(url:, body:, token: nil)
         def body(file, **options)
           deprecation_warning(options)
 
+          definition = Definition.new(file, import_external_references: options[:'import-external-references'])
+          definition.prepare!
+
           compact(
             {
-              definition: prepare_file(file, options),
+              definition: definition.content,
```

Comment:
With this change, `prepare_file` become deadcode. Either if I miss something (review still in progress), method should be removed, and specs should be cleaned:
> `allow(command).to receive(:prepare_file).and_return("body")`

Reply:
Nice catch, thanks! Fixed in a huge (but useful) rebase!